DXRando/DeusEx/Classes/Player.uc

```diff
@@ -203,6 +212,11 @@ function bool DXReduceDamage(int Damage, name damageType, vector hitLocation, ou
             skillLevel = SkillSystem.GetSkillLevelValue(class'SkillEnviro');
             newDamage *= 0.75 * skillLevel;
         }
+        else // passive enviro skill still gives some damage reduction
+        {
+            skillLevel = SkillSystem.GetSkillLevelValue(class'SkillEnviro');
+            newDamage *= (skillLevel + 1)/2;
+        }
```

Comment:
the passive buff to enviro skill here means if you get the vanilla advanced skill level of 0.5, then this would mean `newDamage *= 0.75` aka a 25% reduction of damage